internal/display/apis.go

```diff
@@ -67,13 +69,12 @@ func (r *Renderer) ApiUpdate(api *management.ResourceServer) {
 }
 
 func makeApiView(api *management.ResourceServer) *apiView {
-	scopes := len(api.Scopes)
 
 	return &apiView{
 		ID:         auth0.StringValue(api.ID),
 		Name:       auth0.StringValue(api.Name),
 		Identifier: auth0.StringValue(api.Identifier),
-		Scopes:     auth0.IntValue(&scopes),
+		Scopes:     auth0.StringValue(truncateScopes(api.Scopes)),
```

Comment:
This results in the list of scopes being too large for the table view:

<img width="1302" alt="Screen Shot 2021-03-19 at 21 13 02" src="https://user-images.githubusercontent.com/5055789/111853035-07db1f80-88f8-11eb-9b4c-15d23424e8af.png">

For now, we can leave the scopes count on the table view. Apps use a different view struct to render the table view: https://github.com/auth0/auth0-cli/blob/main/internal/display/apps.go#L128

Reply:
Good catch! Updated.